views/experience.hbs

```diff
@@ -3,29 +3,45 @@
         <div class="text-box text-box__experience u-margin-left-medium">
             <h1 class="heading-primary heading-primary__experience u-margin-bottom-small">Experience:</h1>
             <div class="experience">
-                <div class="progress-bar progress-bar__1">
-                    <h2 class="name">Illustrator</h2>
-                </div>
-                <div class="progress-bar progress-bar__2">
-                    <h2 class="name">Photoshop</h2>
-                </div>
-                <div class="progress-bar progress-bar__3">
-                    <h2 class="name">JS</h2>
-                </div>
-                <div class="progress-bar progress-bar__4">
-                    <h2 class="name">Node.js</h2>
-                </div>
-                <div class="progress-bar progress-bar__5">
-                    <h2 class="name">Git</h2>
-                </div>
-                <div class="progress-bar progress-bar__6">
-                    <h2 class="name">Sass/Scss</h2>
-                </div>
-                <div class="progress-bar progress-bar__7">
-                    <h2 class="name">Css</h2>
-                </div>
-                <div class="progress-bar progress-bar__8">
-                    <h2 class="name">Html</h2>
+                <div class="progress-bar ">
+                    <div class="progress-bar__1">
+                        <h2 class="name">Illustrator</h2>
+                    </div>
+                </div>
+                <div class="progress-bar">
+                    <div class="progress-bar__2">
```

Comment:
I would set width directly here on element style="width: 35%" and for like this for all